api/v01/vrp.rb

```diff
@@ -75,10 +75,32 @@ class Vrp < APIBase
           desc 'Submit VRP problem', {
 
             nickname: 'submit_vrp',
-            success: VrpResult,
-            failure: [
-              { code: 400, message: 'Bad Request', model: ::Api::V01::Status }
-            ],
+            success: [{
+              code: 200,
+              message: 'The VRP has been processed synchronously',
+              model: VrpSyncJob
+            }, {
+              code: 201,
+              message: 'The VRP has been placed in a queue to be processed',
+              model: VrpAsyncJob
+            }],
+            failure: [{
+                code: 400,
+                message: 'Bad Request',
+                model: ::Api::V01::Status
+              }, {
+                code: 401,
+                message: 'Unauthorized',
+                model: ::Api::V01::Status
+              }, {
+                code: 402,
+                message: 'Subscription expired',
```

Comment:
We can add a message like 'Please contact support to renew your subscription.' , what do you think?

Also should we give an email <xxx@mapotempo.com> and/or phone <+33xxx> with these messages (in addition to or instead of just saying support/sales)?